packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -121,24 +70,15 @@ const UserForm: React.FC<UserFormProps> = (props: UserFormProps) => {
     { label: 'Yes', value: true },
     { label: 'No', value: false },
   ];
-  const { Option } = Select;
+
   React.useEffect(() => {
-    fetchRequiredActions(keycloakBaseURL, setUserActionOptions, serviceClass);
-  }, [keycloakBaseURL, serviceClass]);
+    fetchRequiredActions(keycloakBaseURL, setUserActionOptions);
+  }, [keycloakBaseURL]);
+
   React.useEffect(() => {
     setRequiredActions(initialValues.requiredActions ? initialValues.requiredActions : []);
   }, [initialValues.requiredActions]);
 
-  /** Update form initial values when initialValues prop changes, without this
-   * the form fields initial values will not change if props.initiaValues is updated
-   * **/
-  React.useEffect(() => {
-    form.setFieldsValue({
```

Comment:
why remove this?

Reply:
because we no longer need to set the active value after mounting the component, i have changed the way the default props were being passed into the component.

Before : data without active was passed after component mount and then the active value was set and the component rerender

Now : the active value come with with the data right when the component is being mounted thus no need to set the active value again and thus prevents the extra rerender

Reply:
Have you checked if this change affects the `Manage Account` page?

Reply:
yes i have tested the implementation of this change